src/main/scala/com/simplytyped/Antlr4Plugin.scala

```diff
@@ -56,7 +56,7 @@ object Antlr4Plugin extends Plugin {
     sourceDirectory <<= (sourceDirectory in Compile) {_ / "antlr4"},
     javaSource <<= (sourceManaged in Compile).apply(_ / "antlr4"),
     managedClasspath <<= (configuration, classpathTypes, update) map Classpaths.managedJars,
-    antlr4Version := "4.5.1",
+    antlr4Version := "4.5.2",
```

Comment:
@ihji my apologies, `4.5.2-1` is actually the latest version. could you please update and publish again? so sorry for the lapse :-(